libs/sdk-ui-dashboard/src/presentation/filterBar/attributeFilter/AttributeFilterParentFilteringContext.tsx

```diff
@@ -72,7 +72,7 @@ export const AttributeFilterParentFilteringProvider: React.FC<
         onDisplayFormChange,
         onConfigurationClose: onDisplayFormClose,
         displayFormChangeStatus,
-    } = useDisplayFormConfiguration(currentFilter, attributes);
+    } = useDisplayFormConfiguration(currentFilter, attributes ?? []);
```

Comment:
Pls memoize attributes check for undefined when attributes ?? [] will be undefined its new instance every render  this is problematic construction

Reply:
Fixed.